source/type/pb_converter.cpp

```diff
@@ -352,21 +353,16 @@ void PbCreator::Filler::fill(NAV* nav_object, PB* pb_object) {
 template<typename NAV, typename F>
 void PbCreator::Filler::fill_with_creator(NAV* nav_object, F creator) {
     if (nav_object == nullptr) { return; }
-    DumpMessageOptions new_dump_options{dump_message_options};
-    new_dump_options.dump_line_section = DumpLineSectionMessage::No;
-    copy(depth-1, new_dump_options).fill_pb_object(nav_object, creator());
```

Comment:
Not sure of the implications of that. @xlqian what do you think?

Reply:
I checked this line, there shouldn't be any problems if it's removed